nlp/build-model-worker-on-demand/src/main/kotlin/WorkerOnDemandVerticle.kt

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright (C) 2017/2020 e-voyageurs technologies
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package ai.tock.nlp.build.ondemand
+
+import io.vertx.core.AbstractVerticle
+import io.vertx.core.Handler
+import mu.KotlinLogging
+import java.time.LocalTime
+import java.util.concurrent.TimeUnit
+
+class WorkerOnDemandVerticle(
+    private val workerOnDemandType: WorkerOnDemandType,
+    private val buildType: String,
+    private val delayBetweenJob: Long,
+    private val timeFrame: List<Int>
+) : AbstractVerticle() {
+
+    private val logger = KotlinLogging.logger {}
+
+    private var handler: Handler<Long>? = null
+
+    private var workerOnDemand: WorkerOnDemand? = null
+
+    override fun start() {
+        logger.info("Starting WorkerOnDemandVerticle for $buildType ...")
+
+        workerOnDemand = WorkerOnDemandProvider.provide(
+            type = workerOnDemandType,
+            properties = mapOf("TOCK_BUILD_TYPE" to buildType) + workerProperties()
```

Comment:
Create constants for worker properties names and group them (with those used in workerProperties() method) so that we have a complete list of all the properties in one place

Reply:
Propertues "TOCK_BUILD_TYPE", "tock_build_worker_mode" and "tock_build_worker_verticle_enabled" must not be defined by the user.

Users can pass any properties on AWS BATCH job by prefixing property name with "tock_worker_ondemand"

Reply:
I mean i find better to list the parameters defined in our code together (not those that users can pass)
For example: 
```
BUILD_TYPE= "TOCK_BUILD_TYPE"
WORKED_MODE = "tock_build_worker_mode"
WORKER_VERTICLE_ENABLED = "tock_build_worker_verticle_enabled"
```

define also for example a constant `DEFAULT_WORKED_MODE = "COMMAND_LINE"`
also the prefix PREFIX = "tock_worker_ondemand"
so that we don't have raw strings in our codeand all static parameters that we define can be found in one place